blocklist-client/src/api/handlers.rs

```diff
@@ -0,0 +1,42 @@
+use crate::client::client;
+use crate::common::{Error, ErrorResponse};
+use crate::config::RiskAnalysisConfig;
+use reqwest::Client;
+use std::convert::Infallible;
+use tracing::error;
+
+use warp::{http::StatusCode, Rejection, Reply};
+
+pub async fn check_address_handler(
+    address: String,
+    client: Client,
+    config: RiskAnalysisConfig,
+) -> Result<impl Reply, Rejection> {
+    match client::check_address(client, &config, &address).await {
+        Ok(value) => Ok(warp::reply::json(&value)),
+        Err(_) => Err(warp::reject::custom(Error::AddressNotFound)),
+    }
+}
```

Comment:
I think we should take the config and address by reference here.

```suggestion
pub async fn check_address_handler(
    address: &str,
    client: Client,
    config: &RiskAnalysisConfig,
) -> Result<impl Reply, Rejection> {
    match client::check_address(client, config, address).await {
        Ok(value) => Ok(warp::reply::json(&value)),
        Err(_) => Err(warp::reject::custom(Error::AddressNotFound)),
    }
}
```

Reply:
Nvm, I assume warp passes these by value already.